src/hivpy/population.py

```diff
@@ -81,6 +97,9 @@ def get(self, attribute_name):
         """Get the value of the named attribute at the current date."""
         return self.attributes[attribute_name](self.data)
 
+    def _update_HIV_status(self):
+        """Update HIV status for new transmissions in the last time period"""
+
```

Comment:
Similarly, this could go in the sexual behaviour module, and called from `evolve`? Or perhaps it will end up calling a series of update/util functions from other modules, in which case it's useful as a group here.